site/content/getting-started/index.md

```diff
@@ -72,7 +74,7 @@ Note that ZAP requires Java 11+ in order to run. The macOS installer includes an
 Once the installation is complete, launch ZAP and read the license terms. Click **Agree** if you accept the terms, and ZAP will finish installing, then ZAP will automatically start.
 
 ###### macOS
-OWASP ZAP is currently not a verified developer with Apple. On macOS, you will see a message like: 
+ZAP is currently not a verified developer with Apple. On macOS, you will see a message like: 
 > “OWASP ZAP.app” cannot be opened because the developer cannot be verified.
```

Comment:
Does this need to be updated?

Reply:
No, because we havnt changed the code yet :)